st2api/st2api/controllers/v1/packs.py

```diff
@@ -284,6 +289,10 @@ class PacksController(BasePacksController):
     views = PackViewsController()
     index = PacksIndexController()
 
+    def __init__(self):
+        super(PacksController, self).__init__()
+        self.get_one_db_method = self._get_by_ref_or_id
```

Comment:
That's how RBAC decorator for "get one" works - it uses method defined on the constructor / class.

It's also defined in other places, but it defaults to "get by name or id" which is not what we want here.